.devcontainer/Dockerfile

```diff
@@ -3,21 +3,35 @@ FROM mcr.microsoft.com/devcontainers/cpp:ubuntu-24.04
 RUN apt-get --yes -qq update \
  && apt-get --yes -qq upgrade \
  && apt-get --yes -qq install build-essential \
-                      git cmake clangd gcc g++ \
+                      git cmake gcc g++ \
                       python3-dev python3-numpy python3-matplotlib python3-pip \
-                      sphinx-doc python3-sphinx-rtd-theme python3-sphinxcontrib.bibtex python3-sphinx-copybutton \
                       libopenmpi-dev \
                       libhdf5-mpi-dev \
  && apt-get --yes -qq clean \
  && rm -rf /var/lib/apt/lists/*
 
-WORKDIR /home/ubuntu
-USER ubuntu
+# install Clang 19.x
+RUN wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key| apt-key add - && \
+ echo "deb http://apt.llvm.org/noble/ llvm-toolchain-noble-19 main" > /etc/apt/sources.list.d/llvm.list
+
+RUN apt-get clean && apt-get update -y && \
+ apt-get install -y --no-install-recommends clang-19 llvm-19 libomp-19-dev libclang-rt-19-dev clangd-19 && \
```

Comment:
@chongchonghe `clangd` should still be installed. It just uses the more recent version directly from the LLVM apt repository.